source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1682,8 +1682,10 @@ export class CardElementPeer extends DesignerPeer {
             }
         }
 
+        let itemCount: number = 0;
         if (this.cardElement instanceof Adaptive.CardElementContainer) {
-            for (var i = 0; i < this.cardElement.getItemCount(); i++) {
+            itemCount = this.cardElement.getItemCount();
+            for (var i = 0; i < itemCount; i++) {
```

Comment:
I know this isn't your issue, but while we're here, we should switch `var` to `let` (`var` scopes variables to the containing function)

Reply:
@paulcam206 good call. there are 43 places with `for (var`. Should I fix them all in this PR or another PR

Reply:
I will create a separate PR to address the var problem